sshuttle/methods/nat.py

```diff
@@ -50,6 +50,12 @@ def _ipm(*args):
         _ipt('-I', 'OUTPUT', '1', *args)
         _ipt('-I', 'PREROUTING', '1', *args)
 
+        # Firstly we always skip all LOCAL addtrype address, i.e. avoid
+        # tunnelling the traffic designated to all local TCP/IP addresses.
+        _ipt('-A', chain, '-j', 'RETURN',
+                '-m', 'addrtype',
+                '--dst-type', 'LOCAL')
```

Comment:
It looks like this is breaking use of local DNS servers. Previos versions were working by matching only TCP traffic. Is matching of non-TCP intentional here?